tests/regression/01-cpa/74-rand.c

```diff
@@ -0,0 +1,13 @@
+//PARAM: --enable ana.int.interval
+#include <stdio.h>
+#include <stdlib.h>
+#include <goblint.h>
+
+int main () {
+  int r = rand();
+
+  __goblint_check(r >= 0);
+  __goblint_check(r <= RAND_MAX);
```

Comment:
`r` has the abstract value `ID.starting IInt Z.zero` which isn't sufficient to prove that it's at most `RAND_MAX`. This accidentally passes only when `RAND_MAX` happens to be `INT_MAX` but that's not required. In all other cases this test will spuriously fail.

Reply:
I think `RAND_MAX` is `INT_MAX` on all practical systems though.

Reply:
We shouldn't be making such an incorrect assumption though, especially since it's explicitly implementation-defined by the standard. Even if it's true in practice, then this check isn't really checking anything that Goblint doesn't trivially already know by default. So it's also misleading as it appears like we handle `RAND_MAX`.

The safest thing would be to just remove this check because it adds nothing and just makes us prone to opam CI failures on some less common architectures, machines and distributions. Otherwise we should handle `RAND_MAX` properly, but that's a pain (having to do something machdep-like to preprocess a tiny program with `RAND_MAX` to find out what it expands to).

Reply:
I'll remove this test.